base/interactiveutil.jl

```diff
@@ -134,30 +134,49 @@ if Sys.isapple()
     end
     clipboard() = read(`pbpaste`, String)
 
-elseif Sys.islinux()
+elseif Sys.islinux() || Sys.KERNEL === :FreeBSD
     _clipboardcmd = nothing
+    const _clipboardcmds = Dict(
+        :copy => Dict(
+            :xsel  => Sys.islinux() ?
+                `xsel --nodetach --input --clipboard` : `xsel -c`,
+            :xclip => `xclip -silent -in -selection clipboard`,
+        ),
+        :paste => Dict(
+            :xsel  => Sys.islinux() ?
+                `xsel --nodetach --output --clipboard` : `xsel -p`,
+            :xclip => `xclip -quiet -out -selection clipboard`,
+        )
+    )
     function clipboardcmd()
         global _clipboardcmd
         _clipboardcmd !== nothing && return _clipboardcmd
         for cmd in (:xclip, :xsel)
             success(pipeline(`which $cmd`, DevNull)) && return _clipboardcmd = cmd
         end
-        error("no clipboard command found, please install xsel or xclip")
+        pkgs = @static if Sys.islinux()
+            "xsel or xclip"
+        elseif Sys.KERNEL === :FreeBSD
```

Comment:
I don't think `@static` will resolve the `elseif` yet

Reply:
support of `elseif` block is added in #24787